ipld/hamt/fuzz/Cargo.toml

```diff
@@ -9,20 +9,15 @@ edition = "2021"
 cargo-fuzz = true
 
 [dependencies]
-libfuzzer-sys = "0.4"
-arbitrary = { version = "1.1", features = ["derive"] }
-ahash = "0.7.6"
+libfuzzer-sys = { workspace = true }
+arbitrary = { workspace = true, features = ["derive"] }
+ahash = { workspace = true }
 
-fvm_ipld_hamt = { path = ".." }
-fvm_ipld_blockstore = { path = "../../blockstore" }
-
-
-# Prevent this from interfering with workspaces
-[workspace]
-members = ["."]
```

Comment:
Ah.... I _think_ this may have been why we didn't include this in the workspace. We need to check the `cargo tree` (and enabled features) in the workspace itself to make sure we're not "unifying" them. I'll go ahead and do that.

Reply:
Basically, look up "feature unification in rust" if you want to have bad dreams.

Reply:
Hm. Actually... this seems fine? I _thought_ I saw the arbitrary feature/crates getting included in the main dependency tree... but now I'm not seeing that.